addons/l10n_tr_nilvera_einvoice/views/account_journal_dashboard_views.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_journal_dashboard_kanban_view" model="ir.ui.view">
+        <field name="name">account.journal.dashboard.kanban</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.account_journal_dashboard_kanban_view" />
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//kanban" position="inside">
+                    <field name="is_nilvera_journal" invisible="1"/>
+                    <field name="l10n_tr_nilvera_api_key" invisible="1"/>
+                </xpath>
+
+                <xpath expr="//t[@id='account.JournalBodySalePurchase']//div" position="inside">
+                    <t t-if="record.l10n_tr_nilvera_api_key.raw_value">
```

Comment:
invisible ? Maybe not needed because the journal will not have a l10n_tr_nilvera_api_key on not turkish company :thinking: